Shared/Samples/Orbit camera around object/OrbitCameraAroundObjectView.swift

```diff
@@ -0,0 +1,177 @@
+                }
+        } else {
+            button
+                .sheet(isPresented: $settingsSheetIsPresented, detents: [.medium]) {
+                    settingsContent
+                }
+        }
+    }
+}
+
+private extension OrbitCameraAroundObjectView {
+    /// The camera and plane settings for the sample.
+    struct SettingsView: View {
+        /// The view model for the sample.
+        @ObservedObject var model: Model
+        
+        /// The action to dismiss the view.
+        @Environment(\.dismiss) private var dismiss: DismissAction
+        
+        /// The pitch of the plane in the scene.
+        @State private var planePitch = Measurement(value: 0, unit: UnitAngle.degrees)
+        
+        /// The heading offset of the camera controller.
+        @State private var cameraHeading = Measurement(value: 0, unit: UnitAngle.degrees)
+        
+        var body: some View {
+            NavigationView {
+                List {
+                    VStack {
+                        Text("Camera Heading")
+                            .badge(
+                                Text(cameraHeading, format: .measurement(width: .narrow))
+                            )
+                        
+                        Slider(value: $cameraHeading.value, in: -45...45, step: 1)
+                            .onChange(of: cameraHeading.value) { newValue in
+                                model.cameraController.cameraHeadingOffset = newValue
+                            }
+                    }
+                    
+                    VStack {
+                        Text("Plane Pitch")
+                            .badge(
+                                Text(planePitch, format: .measurement(width: .narrow))
+                            )
+                        
+                        Slider(value: $planePitch.value, in: -90...90, step: 1)
+                            .onChange(of: planePitch.value) { newValue in
+                                model.planeGraphic.setAttributeValue(newValue, forKey: "PITCH")
+                            }
+                    }
+                    
+                    Toggle(
+                        "Allow Camera Distance Interaction",
+                        isOn: $model.cameraController.cameraDistanceIsInteractive
+                    )
+                    .disabled(model.cameraController.autoPitchIsEnabled)
+                    .toggleStyle(.switch)
+                }
+                .navigationTitle("Settings")
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .confirmationAction) {
+                        Button("Done") {
+                            dismiss()
+                        }
+                    }
+                }
+            }
+            .navigationViewStyle(.stack)
+            .onAppear {
+                planePitch.value = model.planeGraphic.attributes["PITCH"] as? Double ?? 0
```

Comment:
We know it has a value
```suggestion
                planePitch.value = model.planeGraphic.attributes["PITCH"] as! Double
```